src/assets/jss/breadTableStyle.js

```diff
@@ -0,0 +1,19 @@
+const breadTableStyle = theme => ({
+  buttonCell: {
+    width: '150px',
```

Comment:
buttonCell 150px? lijkt iets dat 44px moet hebben ...

Reply:
aha oke twee buttons in 1 .. noem het mss buttonsCell en wss iets kleiner is iets beter dan (voor mobile)

Reply:
buttonsCell oke, maar veel kleiner kan het niet en het lijkt geen effect te hebben op de mobile versie (150px = ~2x een button)

Reply:
een button is 44px x 44px, twee buttons is 88px, 150px is duidelijk te groot ...

Reply:
door de vreemde padding daar (.MuiTableCell-root-272:last-child) moet er nog 24px bij